### DISCUSSION TITLE
how to send .js .mjs .css files from hooks folder

### DISCUSSION THREAD

**Discussion by gokulgseven** [pocketbase/pocketbase#4563] [Q&A] [answered]
routerAdd("GET", "_/admin/static/config.mjs", (c) => {
      try{
          return c.fileFS("config.mjs"), $filesystem.fileFromPath(`${__hooks}/_.admin/static/config.mjs`)
      }catch(error){
          console.log(error, "error in mjs file")
      }
  })
  
  my use case is that if i send the css and js files required from hooks folder i can just copy a single folder , to copy the complete functionality from one deployment to another

**Comment by ganigeorgiev** [comment, marked as answer]
You could check https://pocketbase.io/docs/js-routing/#serving-static-files.

The folder can be created anywhere (_note that the path is relative to the executable, not where the hooks file is_). I'm not entirely sure that I understand the above code snippet, nor what is the expected serving path but here is one example:

```js
// if we have a file `pb_hooks/yourDir/test.txt`, it could be accessed via `GET /assets/test.txt`
routerAdd("GET", "/assets/*", $apis.staticDirectoryHandler(`${__hooks}/yourDir`, false))
```

**Reply by gokulgseven** [reply]
worked perfectly
